source/meetings/general/2023-01-24.rst

```diff
@@ -0,0 +1,43 @@
+General Meeting 09:00 18-Sep-2023
+=================================
+
+Held at City Hall, Cardiff during PyCon UK 2023 and chaired by Owen Campbell.
+
+Trustees Present
+----------------
+
+Kristian Glass, Becky Smith, Owen Campbell, Vince Knight, Ann Barr
+
+
+Report
+------
+
+Owen Campbell presented the report to the members. Owen presented the
+achievements for the year:
+
+- PyCon UK ran again in 2022 after being postponed during the pandemic.
+  This is a notable achievement given the difficulties related to the pandemic.
+- The UKPA owns UKPA Trading Limited and for the first time the company was
+  ready for the conference. There is still some more progress to be made:
```

Comment:
```suggestion
  operationally ready for the conference. There is still some more progress to be made:
```

```suggestion
  operationally ready for the conference. There is still some more progress to be made:
```

nitpick: (And apologies for all the nitpicks!) I prefer "operationally ready" over just "ready" to provide a stronger indication that it was things like "we have not yet run the gauntlet of opening a business bank account" rather than e.g. any other lack of enthusiasm/preparedness on our parts(!)